themes/default/content/blog/benchmarking-python-performance/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "Benchmarking Python Performance"
+date: 2023-07-31
+meta_desc: "Benchmarking and improving the performance of Pulumi Python programs."
+meta_image: meta.png
+
+authors:
+    - robbie-mckinstry
```

Comment:
You may want to change this value, or add another author's name

Reply:
Done.